script/Ciel/Day 14/14_01B_01_CIEL14B_1B.txt

```diff
@@ -564,17 +642,24 @@ I ended up killing the city that I swore to protect.
 [sha:a3724f52147f8fc629b4b6ac2598fa86957b4e9e]{
 -- Page 71, Offset 38551.
 -- ……足は動かず、逃げ場所すら思いつかない。
-...I can't move my legs, and I can't even think of a place to run to.
+I can't move my legs...
+ I can't even think where I could possibly escape to.
 }
 [sha:0c2e730106db741ef1bc468075f02e49ef85698a]{
 -- Page 71, Offset 38552.
 -- どのみち、何処に逃げても。
-But I realized no matter where I run,
+// Do people say やがるのか to themselves? Or are these last 2 lines Roa?
+// Flopped for flow
+%{i}
+Do you finally understand it now?
+%{/i}
 }
 [sha:0538c3170cf0f4d90a99a9321a1beda55ef0b7ca]{
 -- Page 71, Offset 38553.
 -- もう、絶対に殺されるって理解してやがるのか。
-I will definitely end up being killed.
+%{i}
+Whatever happens――no matter how far you run――you will be killed.
+%{/i}
 }
 [sha:a4d74217254f58bea6a1318458193a44421fa0e8]{
 -- Page 72, Offset 38554.
```

Comment:
What weird progression.

38555
>I hear her voice.

38556
>And that's enough to dispel the fear gripping me.

alt
38555
>Upon hearing that voice———

38556
>The fear gripping me is dispelled.

Reply:
Yeah it's because that line is shared with the other file, so it got updated when I rebased but the surroundings didn't